### PR TITLE
Update Japanese translation(1.16)

### PR DESCRIPTION
**Pull request by koh-gh**
- Added new translation
- Unified the disjointed translations
- Changed the translation of 'Manyullyn'

### CODE REVIEW

File: src/main/resources/assets/tconstruct/lang/ja_jp.json [hunk 1]
```diff
@@ -1,1108 +1,1181 @@
...
-  "fluid.tconstruct.flowing_molten_knightslime": "溶騎士スライム",
-  "item.tconstruct.molten_knightslime_bucket": "溶騎士スライム入りバケツ",
+  "block.tconstruct.earth_slime_fluid": "液状スライム",
+  "fluid.tconstruct.earth_slime": "液状スライム",
+  "fluid.tconstruct.flowing_earth_slime": "流れる液状スライム",
+  "item.tconstruct.earth_slime_bucket": "液状スライム入りバケツ",
+
+  "block.tconstruct.sky_slime_fluid": "液状スカイスライム",
+  "fluid.tconstruct.sky_slime": "液状スカイスライム",
+  "fluid.tconstruct.flowing_sky_slime": "液状スカイスライム",
```
Comment (karakufire) [inline src/main/resources/assets/tconstruct/lang/ja_jp.json]: 「流れる」を語頭につけ忘れているように見受けられます

File: src/main/resources/assets/tconstruct/lang/ja_jp.json [hunk 2]
```diff
@@ -1,1108 +1,1181 @@
...
+  "item.tconstruct.molten_tungsten_bucket": "溶融したタングステン入りバケツ",
+
+  "block.tconstruct.molten_osmium_fluid": "溶融したオスミウム",
+  "fluid.tconstruct.molten_osmium": "溶融したオスミウム",
+  "fluid.tconstruct.flowing_molten_osmium": "流れる溶融したオスミウム",
+  "item.tconstruct.molten_osmium_bucket": "溶融したオスミウム入りバケツ",
+
+  "block.tconstruct.molten_uranium_fluid": "溶融したウラン",
+  "fluid.tconstruct.molten_uranium": "溶融したウラン",
+  "fluid.tconstruct.flowing_molten_uranium": "溶融したウラン",
```
Comment (karakufire) [inline src/main/resources/assets/tconstruct/lang/ja_jp.json]: 「流れる」を語頭につけ忘れているように見受けられます

**Comment by koh-gh** [comment]
更新しました。コメントありがとうございました。

**Comment by koh-gh** [comment]
Since I added the translations of the book, I changed the title.

File: src/main/resources/assets/tconstruct/lang/ja_jp.json [hunk 3]
```diff
@@ -1,1108 +1,1174 @@
...
   "block.tconstruct.rose_gold_block": "ローズゴールドブロック",
-  "block.tconstruct.rose_gold_block.tooltip": "ビーコンベース可能",
   "item.tconstruct.rose_gold_ingot": "ローズゴールドインゴット",
   "item.tconstruct.rose_gold_nugget": "ローズゴールド塊",
 
-  "block.tconstruct.pig_iron_block": "豚鉄ブロック",
-  "block.tconstruct.pig_iron_block.tooltip": "ビーコンベース可能",
-  "item.tconstruct.pig_iron_ingot": "豚鉄インゴット",
-  "item.tconstruct.pig_iron_nugget": "豚鉄塊",
+  "block.tconstruct.pig_iron_block": "銑鉄ブロック",
```
Comment (CKenJa) [inline src/main/resources/assets/tconstruct/lang/ja_jp.json:115]: 銑鉄はそのままピッグアイアンのほうがわかりやすいかと
1.12.2のときは豚鉄だった気がしますし

File: src/main/resources/assets/tconstruct/lang/ja_jp.json [hunk 4]
```diff
@@ -1,1108 +1,1174 @@
...
   "block.tconstruct.copper_ore": "銅鉱石",
   "block.tconstruct.copper_block": "銅ブロック",
-  "block.tconstruct.copper_block.tooltip": "ビーコンベース可能",
   "item.tconstruct.copper_ingot": "銅インゴット",
   "item.tconstruct.copper_nugget": "銅塊",
 
-  "block.tconstruct.cobalt_ore": "ネザーラックのコバルト原石",
-  "block.tconstruct.cobalt_block": "コバルトのブラック",
-  "block.tconstruct.cobalt_block.tooltip": "ビーコンベース可能",
+  "block.tconstruct.cobalt_ore": "ネザーコバルト原石",
```
Comment (CKenJa) [inline src/main/resources/assets/tconstruct/lang/ja_jp.json]: raw oreではないので原石ではなく鉱石が適切かと

File: src/main/resources/assets/tconstruct/lang/ja_jp.json (hunk 3 above)
Comment (koh-gh) [reply src/main/resources/assets/tconstruct/lang/ja_jp.json:115]: 確かにTiCではPig Ironに豚のイラストを使っていてその意見も理解できるのですが、実際「Pig Iron」の日本語は「銑鉄」であり、ネットを検索すればなぜ「銑鉄」が英語で「Pig Iron」と呼ばれているのか複数のサイトで解説されています。このため、今回は正確な日本語である「銑鉄」にしています。
なぜ「銑鉄」のイラストに豚が使われているのか興味を持つ人が出れば、英語を学ぶ機会にもなるかなと考えています。

File: src/main/resources/assets/tconstruct/lang/ja_jp.json (hunk 4 above)
Comment (koh-gh) [reply src/main/resources/assets/tconstruct/lang/ja_jp.json]: 修正漏れでした。ありがとうございました。

File: src/main/resources/assets/tconstruct/lang/ja_jp.json (hunk 3 above)
Comment (koh-gh) [reply src/main/resources/assets/tconstruct/lang/ja_jp.json:115]: I think "pig iron" is not a fictitious metal like "pig-infused iron", but real metal. And piggy textures are joke.

**Comment by KnightMiner** [comment]
This translation looks relatively up to date. Are you available to get the last few new features for 1.16.5 in? If so we can merge this in for one last 1.16 release with updated translations. I can merge it in without updates, but would be nice for it to be complete

**Comment by koh-gh** [comment]
This translation has been confirmed to work with 3.3.4.335, so you can merge it as it is.

**Comment by koh-gh** [comment]
There seems to be a dev build 1.16.5-3.3.4.336, are there any fixes that affect the translation? If not, this will work.

**Comment by KnightMiner** [comment]
Probably just an automatically made build when Jenkins saw I pushed commits, nothing that is not on curseforge. That said, your changes last month will not include the new features I added this week (couple backports of 1.18 features). Like I said, I can merge it without updates, it will mostly work. Would just be nice to have the new features.

**Comment by koh-gh** [comment]
This includes darkthread, ancient hide, 4 embellishments. Please see https://github.com/SlimeKnights/TinkersConstruct/pull/4707/commits/d81b1c57355d2f2487f657c18e11b908fb7409dc. GitHub shows it as last month, but it was updated yesterday.
But diamond modifier update was lacked. Updated this.

**Comment by KnightMiner** [comment]
Huh, weird. Must be a github bug. Need to do one last test, so should be able to merge it in some time this week.

**Comment by KnightMiner** [comment]
I am merging this to ensure if we do another 1.16 release its included, but at this time another 1.16 release is not guaranteed.

**Pull request merged**